lms/static/js/courseware/accordion_events.js

```diff
@@ -0,0 +1,20 @@
+;(function(define) {
+    'use strict';
+
+    define(['jquery', 'logger'], function ($, Logger) {
+        return function () {
+            $(".accordion-nav").click(function(event) {
+                Logger.log(
+                    "edx.ui.lms.outline.selected",
+                    {
+                        name: "edx.ui.lms.outline.selected",
+                        event_type: "edx.ui.lms.outline.selected",
+                        current_url: window.location.href,
+                        target_url: event.currentTarget.href,
+                        target_name: event.currentTarget.childNodes[1].textContent,
```

Comment:
@jcdyer good call. I added a css class to the display name paragraph element and can pull it more directly via that.